packages/template-typescript-minimal/app/routes.tsx

```diff
@@ -8,14 +8,14 @@ import loadable, {LoadableComponent} from '@loadable/component'
 
 const Home = loadable(() => import('./pages/home'))
 
-const routes = [
-    {
-        path: '/',
-        exact: true,
-        // Type assertion because otherwise we encounter this error:
-        // Exported variable 'routes' has or is using name 'Props' from external module "./app/pages/home" but cannot be named.
-        component: Home as LoadableComponent<unknown>
-    }
+const routes: any = [
```

Comment:
TODO: reverting change before merging

Reply:
Please revert the changes made to the typescript-minimal too.